src/basic/path-util.c

```diff
@@ -782,6 +782,10 @@ static int executable_is_good(const char *executable) {
                                "/dev/null");
 }
 
+int fsck_available(void) {
+        return executable_is_good("/sbin/fsck");
```

Comment:
Please drop `/sbin/`, simply passing `"fsck"` should work.

Reply:
The idea was to mirror `systemd-fsck`: since it explicitly execs `"/sbin/fsck"` without $PATH lookup I figured I should check for exactly that here too (not that it matters in practice, a system which has `fsck` but not in `/sbin/` is kind of broken anyway)